### PR TITLE
Adds  SQL alternative for worksets

### PR DESCRIPTION
**Pull request by Galithil**
Also, comes with a validation script.

### CODE REVIEW

**Comment by Galithil** [comment]
the multiprocessing part still needs testing.

**Comment by Galithil** [comment]
Okay, everything is nice and well, it's faster, better, and stronger than the old one.

**Comment by ewels** [comment]
daft punk

File: scripts/workset_validation.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    lc = lclasses.LimsCrawler(lims, wsp)
+    lc.crawl()
+    ws1 = lclasses.Workset(lims,lc, log)
+    session=get_session()
+    step=session.query(Process).filter_by(luid=ws_id).one()
+    ws2=lclasses.Workset_SQL(session, log,step)
+    diffs=my_comp(ws1.obj, ws2.obj)
+    if diffs:
+        print "\n".join(diffs)
+        #print "##########################"
```
Comment (ewels) [inline scripts/workset_validation.py]: Testing code to remove?

File: scripts/workset_validation.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+        else:
+            diffs.append("key '{}' missing in right dict at depth {}".format(d, path))
+
+
+    return diffs
+
+
+if __name__ == "__main__":
+    print '24-188975'
```
Comment (ewels) [inline scripts/workset_validation.py]: Are these the tests? Maybe a comment to explain what's going on?

**Comment by Galithil** [comment]
It's the validation script, the thing that shows the differences between the two methods for a given setup workset plate. I can change it to be argument based, but I think it's nice to have a test set anyway. what do you think ?

**Comment by ewels** [comment]
ah yeah, i didn't notice the filename, I thought it was part of another script. Yeah that sounds fine then.

**Pull request merged**